### PR TITLE
Completed prep-javascript-objects.

### PR DESCRIPTION
**Pull request by m-Cho1**
prep-javascript-objects

### CODE REVIEW

File: prep-javascript-objects/main.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+var person = {
+  firstName: "James",
+  lastName: "Parker",
+  hobby: "Coding",
+  job: "Web Developer",
+  previousJob: "Student"
+};
```
Comment (thebearingedge) [inline prep-javascript-objects/main.js:7]: You can add new properties to an object later in a program using dot notation.

```js
person.job = 'Accountant';
```

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-javascript-objects/main.js (hunk 1 above)
Comment (m-Cho1) [reply prep-javascript-objects/main.js]: prep-javascript-objects

Comment (m-Cho1) [reply prep-javascript-objects/main.js]: prep-javascript-objects
edited!

**Review by m-Cho1 — COMMENTED** [review]
prep-javascript-objects

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**